server/tests/test_jwt_authentication.py

```diff
@@ -1,6 +1,3 @@
-from datetime import timedelta
-from time import sleep
```

Comment:
We should be running flake8 to detect these errors

Reply:
Done in #1931.